lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -3131,7 +3131,7 @@ public function addEntityListener($eventName, $class, $method)
      * @see getDiscriminatorColumn()
      *
      * @param mixed[]|null $columnDef
-     * @psalm-param DiscriminatorColumnMapping|array{name: string|null, fieldName?: string, type?: string, length?: int, columnDefinition?: string|null, enumType?: class-string<BackedEnum>|null}|null $columnDef
+     * @psalm-param DiscriminatorColumnMapping|array{name: string|null, fieldName?: string, type?: string, length?: int, columnDefinition?: string|null, enumType?: class-string<BackedEnum>|null, options?: array<string, mixed>}|null $columnDef
```

Comment:
:thinking: I don't know why I left this `|array{…}` part in https://github.com/doctrine/orm/pull/10513 but it looks wrong now.

Reply:
https://github.com/doctrine/orm/pull/10602

Reply:
Merged. please address the conflict.